site/app/views/grading/ElectronicGraderView.php

```diff
@@ -1460,49 +1459,49 @@ function display_files($files, &$count, $indent, &$return, $filename) {
             $d = 0;
             $first = true;
             $noChange = "";
-            foreach ($question->getMarks() as $mark) {
-
-            	//Makes the mark blue if they're publish marks
-            	if ($mark->getPublish() === 't') {
-            		$is_publish = "is_publish";
-            	}
-     			else {
-     				$is_publish = "";
-     			}
-
-                if ($first === true) {
-                    $first = false;
-                    $noChange = "readonly";
-                    $mark_text = ($question->getDefault() == 0) ? "No Credit" : "Full Credit";
-                }
-                else {
-                    $noChange = "";
-                    $mark_text = $mark->getNote();
-                }
-                $icon_mark = ($mark->getHasMark() === true && $show_graded_info) ? "fa-square" : "fa-square-o";
-                $mark_name = "mark_text_{$c}_{$d}";
-                $return .= <<<HTML
-                    <tr id="mark_id-{$c}-{$d}" name="mark_{$c}" class="{$is_publish}">
-                        <td colspan="1" style="text-align: center; width: 12%; white-space: nowrap;">
-                            <span onclick="selectMark(this);"> <i class="fa {$icon_mark} mark fa-lg" name="mark_icon_{$c}_{$d}" style="visibility: visible; cursor: pointer; position: relative; top: 2px;"></i> </span>
-                            <input name="mark_points_{$c}_{$d}" type="number" step="{$precision}" onchange="fixMarkPointValue(this);" value="{$mark->getPoints()}" min="{$min}" max="{$max}" style="width: 50%; resize:none; min-width: 50px;" {$noChange}>
-                        </td>
-                        <td colspan="3" style="white-space: nowrap;">
-                                <textarea id = "{$mark_name}" name="{$mark_name}" onkeyup="" rows="1" style="width: 90%; resize:none;" oninput="adjustSize(name)" {$noChange}>{$mark_text} </textarea>
-                                <span id="mark_info_id-{$c}-{$d}" onclick="{$break_onclick} saveMark({$c},'{$gradeable->getId()}' ,'{$user->getAnonId()}', {$gradeable->getActiveVersion()}, {$question->getId()}, '{$your_user_id}'); getMarkInfo(this, '{$gradeable->getId()}');"> <i class="fa fa-users icon-got-this-mark"></i> </span>
-                        </td>
-                    </tr>
-HTML;
-                $d++;
-            }
-            $has_mark = false;
+//             foreach ($question->getMarks() as $mark) {
```

Comment:
It was basically duplicate code that was in this class and also the ta-grading.js file, and I had just commented it out for the time being in case it needed to be present again. I'll probably delete it in some future PR.